### ISSUE TITLE
Cannot locate specified Dockerfile: Dockerfile

### ISSUE THREAD

**Issue by aghayeff** [yiisoft/yii2-app-advanced#418]
### Additional info

| Q                | A
| ---------------- | ---
| Yii vesion       |2
| PHP version      |7.2
| Operating system |Mac OS

**Comment by aghayeff** [comment]
After running this command on console, as you described

`docker-compose run --rm backend composer install`

I get an error:
`ERROR: Cannot locate specified Dockerfile: Dockerfile`

It is another Yii Project. i have a backend, frontend directories as yours project. I haven't checked this isse on your project. But I am pretty sure that I need to apply DockerFile. After some researching, I decided to do some edits on docker-compose.yml

I added 2 lines (context, dockerfile):

```
    build:
        context: frontend
        dockerfile: ../vendor/codeception/base/Dockerfile
```

Then I re run command:

`docker-compose run --rm backend composer install`

It worked!
I don't know is it a good way using Dockerfile from my vendor folder?

Or do I need to create my own DockerFile?

 I just need to launch this project on local. Previously, I used Vagrant.
